src/service/xhr-impl.js

```diff
@@ -265,6 +267,23 @@ export class Xhr {
   getCorsUrl(win, url) {
     return getCorsUrl(win, url);
   }
+
+  /**
+   * Sets up and normalizes the FetchInitDef
+   *
+   * @param {?FetchInitDef=} opt_init Fetch options object.
+   * @param {string=} opt_accept The HTTP Accept header value.
+   * @return {!FetchInitDef}
+   */
+  setupInit_(opt_init, opt_accept) {
```

Comment:
To use in the first line of [CachedXhr#fetchJson](https://github.com/ampproject/amphtml/pull/7562/files#diff-f46e3849881976bd269ba22db88d8766R64). I thought it made sense to share it through subclassing rather than exporting, so that it was clear that it isn't a part of `Xhr`'s public API